tests/scripts/thread-cert/v1_2_test_csl_transmission.py

```diff
@@ -125,6 +125,22 @@ def test(self):
         ssed_messages = self.simulator.get_messages_sent_by(SSED_1)
         self.assertIsNotNone(ssed_messages.next_data_poll())
 
+        # Check if SSED is able to resynchronize with the parent after it is gone longer than the timeout
+        self.nodes[LEADER].start()
+        self.simulator.go(config.LEADER_STARTUP_DELAY)
+        self.nodes[SSED_1].set_csl_timeout(8)
+        self.nodes[SSED_1].set_timeout(10)
+        self.simulator.go(2)
+        self.nodes[LEADER].stop()
+        self.simulator.go(25)
+        self.flush_all()
+        self.nodes[LEADER].start()
+        self.simulator.go(config.LEADER_STARTUP_DELAY)
+        self.assertEqual(self.nodes[LEADER].get_state(), 'leader')
+        self.assertEqual(self.nodes[SSED_1].get_state(), 'child')
```

Comment:
In some runs (on my machine), it fails at the ` self.assertEqual(self.nodes[SSED_1].get_state(), 'child')` step.
Seems like we may be checking too early (before child reattaches). Adding some delay seems to address it:
```suggestion
        self.assertEqual(self.nodes[LEADER].get_state(), 'leader')
        self.simulator.go(5)
        self.assertEqual(self.nodes[SSED_1].get_state(), 'child')
```

Reply:
Sure, I'll update and rebase once 7918 is in.